experiment/simulation/cleansingAction.html

```diff
@@ -0,0 +1,63 @@
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="./css/main.css">
+    <title>Cleansing action</title>
+
+</head>
+    <body>
+        <div class="container-fluid links">
+            <nav class="navbar navbar-expand-sm navbar-custom">
+                <ul class="navbar-nav h3">
+                    <!-- Links for other parts of experiment -->
+                    <li class="nav-item"><a class="nav-link" href="./index.html">Structure of soap</a></li>
+                    <li class="nav-item" ><a class="nav-link activeLink" href="#">Cleansing action of soap</a></li>
+                    <li class="nav-item"><a class="nav-link" href="./micelleFormation.html">Formation of micelle</a></li>
+                </ul>
+                <ul class="navbar-nav ml-auto h3">
+                    <li class="nav-item"><a class="nav-link" href="" onclick="window.location.reload()">Reset Experiment</a></li>
+                </ul>
+            </nav>
+        </div>
+        <div class="container justify-content-center main">
+            <div class="grid">
+                <div class="row">
+                    <h1 class = "title">Cleansing Action of soap.</h1>
+                </div>
+                    <!-- Instruction messages about how to perform the experiment -->
+                <div class="row">
+                    <div class="instructionsBox">
+                        <h3 class="instructions">Instructions:</h3>
+                        <div class="procedureMessage">Click on the spoon to add detergent to water</div>
+                    </div>
+                </div>
+            </div>
+        </div>    
+        <!-- The main visual div -->
+        <div class="dropping-objects">
+            <img src="./images/bench.png" class="bench"/>
+            <img src="./images/spoon.png" class="spoon1" onclick="moveSpoon()"/>
+            <img src="./images/Detergent.png" class="detergent1" onclick="moveSpoon()"/>
+            <img src="./images/glassRod.png" class="glassRod" onclick="moveGlassRod('cleansingAction')"/>
+            <img src="./images/clothWithStain.png" class="clothWithStain" onclick="moveClothWithStain()"/> 
+            <img src="./images/plainCloth.png" class="plainCloth"/>
+            <div class="molecularView">
+                <div class="videoMessage">Molecular view of soap molecules in water:</div>  
+                <video class="videos"  width="100%" height="240"  autoplay	muted loop>
+                    <source class="videoSource" src="./images/videos/MolecularView.mp4" type="video/mp4"/>
```

Comment:
What is the size of this video file? Is it necessary to autoplay this? Can we think about lazy loading the video only when the user requests for it?

Reply:
We could do that but then that video will not be in sync with the experiment. The size of the largest video file I used was 254 Kb. If you want, I can implement lazy loading the video.